.github/workflows/lint.yml

```diff
@@ -0,0 +1,33 @@
+name: lint
+
+on:
+  pull_request:
+    branches:
+      - 'build'
+  push:
+    branches:
+      - 'build'
+      - 'renovate/**'
+
+env:
+  node-version: 14.x
+
+jobs:
+  build:
+    name: prettier
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
```

Comment:
I'm deliberately ignoring the submodule that pulls in the Renovate dependencies, as I'm ignoring it in the `.prettierignore` anyways.